src/test/java/com/palantir/docker/compose/execution/DockerTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.docker.compose.execution;
+
+import static org.apache.commons.io.IOUtils.toInputStream;
+import static org.mockito.Matchers.anyVararg;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DockerTest {
```

Comment:
In general we've been trying to use the convention that unit tests classes are named "FooShould" with test methods that read as sentences, e.g. DockerShould call_rm_with_the_force_flag.

Sorry that things are not 100% consistent in the codebase on this front yet, we've been refactoring as we touch things to make them conform and we perhaps need a more concerted effort to provide uniformity.